ui/src/app/cluster-workflow-templates/components/cluster-workflow-template-details/cluster-workflow-template-details.tsx

```diff
@@ -129,8 +129,7 @@ export const ClusterWorkflowTemplateDetails = ({history, location, match}: Route
                         namespace={namespace}
                         name={template.metadata.name}
                         entrypoint={template.spec.entrypoint}
-                        entrypoints={(template.spec.templates || []).map(t => t.name)}
-                        parameters={template.spec.arguments.parameters || []}
```

Comment:
I think we still need these - otherwise how do you submit the workflow template specifying the spec arguments?